omnibus/package-scripts/agent/postinst

```diff
@@ -96,8 +96,119 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
         fi
     fi
 else
-    echo "macOS: Not implemented yet"
-    exit 1
+    # macOS
+
+    # macOS-specific variables
+    OPT_APP_DIR="$INSTALL_DIR/Datadog Agent.app"
+    APP_DIR="/Applications/Datadog Agent.app"
+    CONF_DIR=$INSTALL_DIR/etc
+    RUN_DIR=$INSTALL_DIR/run
+
+    # Let's log the standard outputs of this script
+    LOG_FILE="$LOG_DIR/postinstall.log"
+    mkdir -vp $LOG_DIR
+    exec > $LOG_FILE 2>&1
+
+    # Let's talk to our user installing the Agent a bit
+    echo "# State at the beginning"
+    echo "## Agent version"
+    grep AGENT_VERSION $INSTALL_DIR/agent/config.py || echo "No config.py file (agent 5)"
+    datadog-agent version || echo "No datadog-agent binary version (agent 6)"
+    echo "## $INSTALL_DIR"
+    ls -al $INSTALL_DIR || "No agent installed"
+    echo "## $APP_DIR/Contents/Resources"
+    ls -al "$APP_DIR/Contents/Resources" || echo "No app installed"
+
+    # Determine current user if he is using the Graphical installer
+    INSTALL_USER=$(ps aux | grep "CoreServices/Installer" | grep -v grep | awk '{print $1;}')
```

Comment:
yes that's correct... This logic comes from the Agent 5. I haven't looked around much but I'm not sure there's a better way to check this (since the script itself runs under `root`)